### PR TITLE
[FIX] Implemented logic to handle cases where no diagnosis/assessment exist in the graph

### LINKED ISSUE #328: Handle case where no diagnosis/assessment exist in the graph
Currently, the query tool emits some error (warning?) messages about "no diagnosis options" being fetched (and similarly for assessment) if the data in the graph doesn't contain any assessment/diagnosis data.

This however doesn't actually prevent you from querying the data that _is_ in the graph (e.g., with an empty query). We should probably either remove this error message or make it more informative to indicate that it doesn't necessarily mean there is a problem with the database.

### PR DESCRIPTION
**Pull request by rmanaem**
- Closes #328 

## Checklist

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`) _(see https://neurobagel.org/contributing/pull_requests for more info)_
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Tests pass
- [x] Checks pass

For new features:
- [x] Tests have been added

For bug fixes:
- [ ] There is at least one test that would fail under the original bug conditions.

### CODE REVIEW

File: components/QueryForm.vue [hunk 1]
```diff
@@ -100,11 +100,11 @@
...
-            <hr>
-          </div>
-          <b-row>
-            <h5>Imaging fields</h5>
-          </b-row>
+          <hr>
+        </div>
+        <b-row>
+          <h5>Imaging fields</h5>
+        </b-row>
```
Comment (surchs) [inline components/QueryForm.vue:107]: do you know why this is moving around? thought eslint checks would prevent such stuff?

File: cypress/e2e/APIRequest.cy.js [hunk 2]
```diff
@@ -19,4 +19,34 @@ describe('API request', () => {
     cy.get('[data-cy="submit-query"]').click();
     cy.wait('@call').its('request.url').should('contains', '&min_age=10&max_age=30');
   });
+  it('Intercepts the requests for retrieving options and mocks empty responses to assert over the info toasts', () => {
```
Comment (surchs) [inline cypress/e2e/APIRequest.cy.js]:
```suggestion
  it('Empty responses for diagnosis and Assessment make a toast appear', () => {
```
I think that's clearer, implementation probably less relevant when we are looking for this

File: cypress/e2e/APIRequest.cy.js [hunk 3]
```diff
@@ -19,4 +19,34 @@ describe('API request', () => {
...
+      method: 'GET',
+      url: '/attributes/nb:Assessment',
+    }, emptyAssessmentToolOptions).as('getAssessmentToolOptions');
+    cy.visit('/');
+    cy.wait('@getDiagnosisOptions');
+    cy.contains('#b-toaster-top-right', 'No diagnosis options were available');
+    cy.wait('@getAssessmentToolOptions');
+    cy.contains('#b-toaster-top-right', 'No assessment tool options were available');
+  });
+  it('Intercepts the requests for retrieving options and mocks failed requests to assert over the error toasts', () => {
```
Comment (surchs) [inline cypress/e2e/APIRequest.cy.js]: Failed responses for diagnosis and assessment make an error toast appear

**Review by surchs — APPROVED** [review]
Very nice @rmanaem, tests look good, other stuff makes sense. I left some :cherries: comments for you to look at, but good to go :cook: 

One thing to check before merging: in the staging build I cannot see any toast appear, even though the assessment and diagnosis are both empty. We should make sure this is expected - and then fix this.

File: components/QueryForm.vue (hunk 1 above)
Comment (rmanaem) [reply components/QueryForm.vue:107]: I reckon its because I used `lintfix` to fix issues and since it does formatting too it moved this around.

**Pull request merged**